git_machete/cmd.py

```diff
@@ -1680,26 +1741,26 @@ def fork_point_and_containing_branch_defs(b, use_overrides):
             return fp_sha, containing_branch_defs
 
 
-def fork_point(b, use_overrides):
+def fork_point(b: str, use_overrides: bool) -> Optional[str]:
     sha, containing_branch_defs = fork_point_and_containing_branch_defs(b, use_overrides)
     return sha
 
 
-def config_key_for_override_fork_point_to(b):
+def config_key_for_override_fork_point_to(b: str) -> str:
     return f"machete.overrideForkPoint.{b}.to"
 
 
-def config_key_for_override_fork_point_while_descendant_of(b):
+def config_key_for_override_fork_point_while_descendant_of(b: str) -> str:
     return f"machete.overrideForkPoint.{b}.whileDescendantOf"
 
 
 # Also includes config that is incomplete (only one entry out of two) or otherwise invalid.
-def has_any_fork_point_override_config(b):
+def has_any_fork_point_override_config(b: str) -> bool:
     return (get_config_or_none(config_key_for_override_fork_point_to(b)) or
             get_config_or_none(config_key_for_override_fork_point_while_descendant_of(b))) is not None
 
 
-def get_fork_point_override_data(b):
+def get_fork_point_override_data(b: str) -> Optional[BRANCH_DEF]:
```

Comment:
Actually... that's not a branch def  that's returned (local+remote branch name), that's a pair of SHAs, see e.g. line 1793:
`return to_sha, while_descendant_of_sha`

Reply:
It also boils down to two strings... but I'd still extract a separate type here